scheduler/src/cook/mesos/task.clj

```diff
@@ -297,6 +309,9 @@
                                   (if (:network docker)
                                     (update docker :network cook-network->mesomatic-network)
                                     docker)))
+                        (update :docker
+                                #(clojure.set/rename-keys % {:port-mapping :port-mappings}))
+                        (assign-port-mappings ports-assigned)
```

Comment:
I would have preferred the inlined `update-in` in here as it consistent with the other `update` calls.

Reply:
Done